### ISSUE TITLE
Cannot revert to previous history versions

### ISSUE THREAD

**Issue by studiospring** [penpot/penpot#56]
Even if I click a previous version and press "accept", the latest ("current") version is always highlighted.

**Comment by niwinz** [comment]
Are you sure? I have checked it right now and it works as expected. 

When you clicks to "accept" a new version is created on top of the rest of history entries with the state of the desired version. No history entry is deleted or discarded on this operation.

**Comment by studiospring** [comment]
I see. This prevents forking and version control messiness. However, I would expect selecting and accepting a version number would change the current version number. Is it possible and do you think more intuitive to change the version number of the "current" version to the version number that I selected?

From a UX perspective, it would be nice to have a slider going down the history, so that I can see the changes as a smooth animation and find the version I want faster, instead of click, click click. :D

**Comment by elhombretecla** [comment]
We already think about add a slider for the version control. But now it has a lower priority :)

**Comment by niwinz** [comment]
@studiospring it is imposible with current approach set a version number to the one that you have selected. This is because the version is a strictly incrementing atomic number that is increases with each persistence operation. So the fact of selecting a previous version and **accepting** it, it will perform a new save operation that will imply that the latest version with the selected change will have a new version number.

On the other hand, image if we can make this change, and do that you are proposing! This will make a lot of mess with version numbers because version numbers will repeat, because you will be able to revert to the same version multiple times in the page life. This is not a desired behavior.

In summary, that I intend to say here that this is more complex that it seems at simple view. And need more thought on it before any change.

**Comment by studiospring** [comment]
Thanks! Got it. It sounds like something to revisit at a later date.

**Comment by niwinz** [comment]
In any case, many thanks for open this kind of discussions, they are very positive and allows revisit some parts of the application.
